src/SFA.DAS.Reservations.Web/Controllers/ProviderReservationsController.cs

```diff
@@ -95,7 +95,10 @@ public async Task<IActionResult> Start(uint ukPrn, bool isFromManage)
 
             if (response?.ActiveGlobalRules != null && response.ActiveGlobalRules.Any())
             {
-                return View( "ProviderFundingPaused");
+                var backLink = isFromManage
+                    ? Url.RouteUrl(RouteNames.ProviderManage, new {ukPrn, isFromManage})
+                    : _externalUrlHelper.GenerateDashboardUrl(); 
```

Comment:
you should be able to get a test around this for making sure its setting the route correctly based on whether or not its come from manage
`_urlHelper.Setup(h => h.RouteUrl(It.Is<UrlRouteContext>(c =>
                    c.RouteName.Equals(RouteNames.ProvideManage) &&
                    c.Values.Equals(_routeModel))))
                .Returns(expectedBackUrl);`
then make sure that the backlink is correctly set

Reply:
Updated

Reply:
👍